tests/performance/loadtest/loadtest_test.go

```diff
@@ -330,7 +330,7 @@ var _ = Describe("Application deployment test in Perfronace test EdgeNodes", fun
 			PullImageInAllEdgeNodes(appDeployments)
 		})
 
-		FMeasure("MEASURE_PERF_NODETEST_SINGLE_NODE_1: Create 100 application Deployments, Measure Pod Running time", func(b Benchmarker) {
+		FIt("MEASURE_PERF_NODETEST_SINGLE_NODE_1_WITH_1_REPLICA: Create 100 application Deployments, Measure Pod Running time", func() {
```

Comment:
What's the background of this change?

Reply:
Since `Measure` has been deprecated, there seems no definition nor declaration for `FMeasure` in ginkgo v2. Using `FIt` instead of `FMeasure` is just like replacing `Measure` with `It` as described in the [migration guide](https://onsi.github.io/ginkgo/MIGRATING_TO_V2#removed-measure:~:text=replacing%20Measure%20nodes%20with%20It), though the former is not explictly writen in this guide.